src/modules/dashboard/components/ActionsPage/DetailsWidget/DetailsWidgetUser/DetailsWidgetUser.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Address } from '~types/index';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import MaskedAddress from '~core/MaskedAddress';
+import styles from './DetailsWidgetUser.css';
+import { useUser } from '~data/index';
+import InvisibleCopyableAddress from '~core/InvisibleCopyableAddress';
+
+const displayName = 'dashboard.ActionsPage.DetailsWidget.DetailsWidgetUser';
+
+interface Props {
+  walletAddress: Address;
+}
+
+const DetailsWidgetUser = ({ walletAddress }: Props) => {
```

Comment:
Please refactor this as a `core` component since we're going to use this in many more places